### PR TITLE
feat: batch_create_persistent_job

### PR DESCRIPTION
**Pull request by gryczj**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: batch/create/create_persistent_disk_job.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  }
+
+  callCreateBatchPersistentDiskJob();
+  // [END batch_create_persistent_disk_job]
+}
+
+process.on('unhandledRejection', err => {
+  console.error(err.message);
+  process.exitCode = 1;
+});
```
Comment (subfuzion) [inline batch/create/create_persistent_disk_job.js:154]: Acknowledging that some inconsistency currently exists in our samples, please remove this. It doesn't add value to the sample; an unhandled rejection will terminate the process and set the exitCode and provide a full stack trace. If it's important to handle, then using `try-catch` blocks (or [Promise.prototype.catch()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch)), is more appropriate.

**Review by subfuzion — CHANGES_REQUESTED** [review]
Hi @gryczj Looks good, just one requested change, please. Thanks!

File: batch/create/create_persistent_disk_job.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+main();
```
Comment (subfuzion) [inline batch/create/create_persistent_disk_job.js:156]: Promise returned from main is ignored. You can address both this and the previous comment I made by make the following change:

```javascript
main().catch(err => {
  console.error(err);
  process.exitCode = 1;
});
```

**Review by subfuzion — CHANGES_REQUESTED** [review]

**Comment by gryczj** [comment]
@subfuzion fixed :)

**Review by subfuzion — APPROVED** [review]

**Pull request merged**